test_runner/fixtures/neon_fixtures.py

```diff
@@ -2480,6 +2480,9 @@ def static_proxy(
     # For simplicity, we use the same user for both `--auth-endpoint` and `safe_psql`
     vanilla_pg.start()
     vanilla_pg.safe_psql("create user proxy with login superuser password 'password'")
+    vanilla_pg.safe_psql(
+        "CREATE TABLE endpoints (endpoint_id VARCHAR(255) PRIMARY KEY, allowed_ips VARCHAR(255))"
```

Comment:
nit: can we make this endpoints table in a separate database schema? eg `neon_control_plane.endpoints`